registry.json

```diff
@@ -12,6 +12,20 @@
         "telegram": "AdaStak_Official",                              
         "operatorName": "Will",		                                   
         "github": "wrp0"                                                      
+    },
+    {
+        "ticker": "Z3LDA",                                                      
+        "poolId": "263578cc21e4acdfb86ca47a90fbb9200a203e45975e6fea423fb438",
+        "social": {                                                            
+            "twitter": "Z3ldaP",
```

Comment:
This account has been suspended. If you no longer plan on using it please can you remove from the PR?